src/data/makeDisplayableBalances.ts

```diff
@@ -19,10 +19,19 @@ export function makeDisplayableBalances(
     (memo, balance) => {
       const identifier = getBalanceIdentifier(balance);
       const total = new BigNumber(balance.balance);
-      const sellingLiabilities = new BigNumber(balance.selling_liabilities);
-      const buyingLiabilities = new BigNumber(balance.buying_liabilities);
+
+      let sellingLiabilities = new BigNumber(0);
+      let buyingLiabilities = new BigNumber(0);
       const available = total.minus(sellingLiabilities);
```

Comment:
I don't see the whole context here, do we need to move `available` after the block on lines 27:29? Just making sure we're using the expected value there.

Reply:
Ooh good catch - yes, this should be moved to line 29